Makefile

```diff
@@ -106,7 +106,7 @@ hook/%: ## run post-build hooks for an image
 	DOCKER_REPO="$(OWNER)/$(notdir $@)" \
 	IMAGE_NAME="$(OWNER)/$(notdir $@):latest" \
 	IMAGE_SHORT_NAME="$(notdir $@)" \
-	$(SHELL) $(notdir $@)/hooks/run_hook
```

Comment:
I deleted this line to make it easier to run `make hook/base-notebook` for example.

Reply:
Not needed anymore